doc/sources/gettingstarted/installation.rst

```diff
@@ -80,7 +80,7 @@ with other Python versions and packages. It's optional **but strongly recommende
 
 #. Create the virtual environment named ``kivy_venv`` in your current directory::
 
-       python -m virtualenv kivy_venv
+       python -m venv kivy_venv
```

Comment:
I'm fine to suggest to use the built-in `venv` command instead of the fully featured `virtualenv`, but maybe in that case we should refer to `https://docs.python.org/3/library/venv.html` instead of `https://virtualenv.pypa.io/en/latest/` ?

Reply:
Heya, modified as per your request